api/src/service/domain/workflow/workflowitem_document_download.ts

```diff
@@ -6,6 +6,7 @@ import { ServiceUser } from "../organization/service_user";
 import * as WorkflowitemDocument from "./document";
 import * as Workflowitem from "./workflowitem";
 import * as WorkflowitemDocumentUploaded from "./workflowitem_document_uploaded";
+import { getDocument as getDocumentService } from '../../workflowitem_document_download';
```

Comment:
@galethil where is this import used?

Reply:
@m-janos import is used in jsdoc comment. Since code is using a lot of dependency injections I added some jsdoc links so I will be able to navigate code more quickly to correct place.

Reply:
@galethil all right, interesting trick